src/App.js

```diff
@@ -1,41 +1,24 @@
 import "./App.css";
-import logo from "./logo.png";
-
+import { Navbar } from "./components/Navbar/Navbar";
+import { Routes, Route } from "react-router-dom";
+import {  LandingPage } from "./pages/LandingPage/LandingPage";
+import { ProductPage } from "./pages/ProductPage/ProductPage";
+import { CartPage } from "./pages/CartPage/CartPage";
+import { SignUp } from "./pages/SignUpPage/SignUp";
+import { LoginPage } from "./pages/LoginPage/LoginPage";
+import { WishlistPage } from "./pages/WishlistPage/WishlistPage";
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <Navbar />
+      <Routes>
```

Comment:
You can create a separate component for routes and them import it here.

Reply:
sure, thanks for suggestion

Reply:
done with the changes